Missing files in latest release

Getting this error when i try running my app with the latest version:


> ERROR in node_modules/ag-grid-autocomplete-editor/ag-grid-autocomplete-editor.d.ts:3:48 - error TS2307: Cannot find module './autocompleter/autocomplete' or its corresponding type declarations.
> 
> 3 import { AutocompleteItem, EventTrigger } from './autocompleter/autocomplete';


If I grab the autocompleter folder from v1.4.0 and copy it to the folder it works, but it seems like that folder isn't being included anymore.

Thanks for the great package, and sorry if it's just me doing something weird!

> Getting this error when i try running my app with the latest version:
> 
> > ERROR in node_modules/ag-grid-autocomplete-editor/ag-grid-autocomplete-editor.d.ts:3:48 - error TS2307: Cannot find module './autocompleter/autocomplete' or its corresponding type declarations.
> > 3 import { AutocompleteItem, EventTrigger } from './autocompleter/autocomplete';
> 
> If I grab the autocompleter folder from v1.4.0 and copy it to the folder it works, but it seems like that folder isn't being included anymore.
> 
> Thanks for the great package, and sorry if it's just me doing something weird!

Same with me.

--- a/package.json
+++ b/package.json
@@ -5,6 +5,7 @@
   "deprecated": false,
   "main": "ag-grid-autocomplete-editor.js",
   "files": [
+    "/autocompleter/autocomplete.d.ts",
     "/ag-grid-autocomplete-editor.d.ts",
     "/ag-grid-autocomplete-editor.js",
     "/ag-grid-autocomplete-editor.js.map",

